app/code/Magento/InventorySales/Plugin/CatalogInventory/StockManagement/ProcessRegisterProductsSalePlugin.php

```diff
@@ -94,61 +100,42 @@ public function __construct(
      * @throws LocalizedException
      * @SuppressWarnings(PHPMD.UnusedFormalParameter)
      */
-    public function aroundRegisterProductsSale(StockManagement $subject, callable $proceed, $items, $websiteId = null)
-    {
+    public function aroundRegisterProductsSale(
+        StockManagement $subject,
```

Comment:
Currently, we have plugin on StockManagement, but it should be listening `\Magento\CatalogInventory\Api\RegisterProductSaleInterface` interface instead